src/schema/v2/homeView/HomeViewSection.ts

```diff
@@ -45,6 +48,13 @@ const ArtworksRailHomeViewSectionType = new GraphQLObjectType<
   interfaces: [GenericHomeViewSectionInterface, NodeInterface],
   fields: {
     ...standardSectionFields,
+
+    artworksConnection: {
+      type: artworkConnection.connectionType,
+      args: pageable({}),
+      resolve: (parent, ...rest) =>
+        ARTWORK_RESOLVERS[parent.key](parent, ...rest),
```

Comment:
This bit of runtime polymorphism allows the artworksConnection to decide on-the-fly [which resolver to use](https://github.com/artsy/metaphysics/pull/5865/files#diff-294b344a66a8ab3968329ebdf23ee095eaa754562be805266f228cd2ac4be3dfR88-R92), based on what kind of `key` the parent section has — `NEW_WORKS_FOR_YOU` vs `RECENTLY_VIEWED_ARTWORKS` etc etc.

This is probably the "cleverest" bit in this PR — how do we feel about it?